tests/packages/geckodriver.bats

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bats
+
+setup() {
+  export GECKODRIVER_VERSION="0.16.1"
```

Comment:
I would add a command to clear out the cached download, so we always install from the upstream provider and get notified if they change a URL or something similar.